README.md

```diff
@@ -70,6 +70,15 @@ If you find a problem, please [file a bug](https://github.com/mdn/browser-compat
 We're very happy to accept contributions to this data. Please familiarize yourself
 with the schema and send us a pull request. See also the [Contributing file](https://github.com/mdn/browser-compat-data/blob/master/CONTRIBUTING.md) for more information.
 
+## Browser compatibility tables on MDN
+
+It takes 1-2 weeks for changes in this data to be reflected in MDN's browser compatibility tables. The process is:
+
+1. A pull request is reviewed and merged to master.
+2. A new release of [mdn-browser-compat-data](https://www.npmjs.com/package/mdn-browser-compat-data) is created by MDN staff. This happens every 4-14 days.
+3. A new image of [Kumascript](https://github.com/mdn/kumascript), which includes the BCD release, is built and deployed to production. This happens within a day of the npm package release.
+4. The MDN page using the data is regenerated. For newly converted pages, a staff member switches to the [{{Compat}](https://github.com/mdn/kumascript/blob/master/macros/Compat.ejs) macro, and re-checks the conversion. For updates to converted pages, a logged-in MDN user force-refreshes the page to regenerate it.
```

Comment:
I would suggest mentioning that the force-refresh feature is activated using **`Ctrl`**+**`Shift`**+**`R`** <code>\*\*\`Ctrl\`\*\*+\*\*\`Shift\`\*\*+\*\*\`R\`\*\*</code>.

Reply:
It's Command-Shift-R on FF for MacOS. If there's a good webpage for "how to force-refresh", I'd link to that.

Reply:
I’ve learnt about it through [a tweet](https://twitter.com/Sphinx_Twitt/status/971114280815923208).

Anyway, it actually just requires an [override cache reload](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_navigation) in [the browser while logged in](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Tools/Page_regeneration).

Reply:
This might be useful https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache#Bypassing_cache